css/css-grid/alignment/grid-container-baseline-001.html

```diff
@@ -110,27 +110,27 @@
   <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="75"></div>
 </div>
 
-<div class="wrapper" style="writing-mode: vertical-rl;">
-  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="65"></div>
+<div class="wrapper" style="writing-mode: vertical-rl; text-orientation: sideways;">
+  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="75"></div>
   <div class="grid">
     <div class="i1"></div>
     <div class="i2" data-offset-x="75"></div>
     <div class="i3"></div>
   </div>
-  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="65"></div>
+  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="75"></div>
 </div>
 
-<div class="wrapper" style="writing-mode: vertical-rl;">
-  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="65"></div>
+<div class="wrapper" style="writing-mode: vertical-rl; text-orientation: sideways;">
+  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="75"></div>
   <div class="grid">
     <div class="i1"></div>
     <div class="i2" style="writing-mode: horizontal-tb;" data-offset-x="75"></div>
     <div class="i3"></div>
   </div>
-  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="65"></div>
+  <div style="display: inline-block; width: 20px; height: 10px; background: black" data-offset-x="75"></div>
 </div>
 
-<div class="wrapper" style="writing-mode: vertical-rl; text-orientation: sideways;">
+<div class="wrapper" style="writing-mode: vertical-rl; orientation: sideways;">
```

Comment:
What are you changing here? Why do you `change text-orientation` by `orientation`, this doesn't look right.

Reply:
sure, thanks for the review.